### PR TITLE
office-tool-plus: Add version 8.3.10.7

### PR DESCRIPTION
**Pull request by ClassicDarkChocolate**
[office-tool-plus](https://github.com/YerongAI/Office-Tool) is a powerful and useful tool for Office deployments.

According to Sandboxie, it store its configuration in registry.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/office-tool-plus.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "Office Tool Plus.exe",
+            "Office Tool Plus"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/YerongAI/Office-Tool"
+    },
+    "autoupdate": {
+        "url": "https://github.com/YerongAI/Office-Tool/releases/download/v$version/Office_Tool_with_runtime_v$version.7z",
+        "hash": "$basename.*?$sha256"
```
Comment (issaclin32) [inline bucket/office-tool-plus.json:20]:
```suggestion
        "hash": {
            "url": "https://github.com/YerongAI/Office-Tool/releases/latest",
            "regex": "(?sm)$basename.*?$sha256"
        }
```

Comment (issaclin32) [reply bucket/office-tool-plus.json:20]: This `autoupdate.hash` is actually not valid. (although PR check passed)
I will try to fix PR check later.

Result of checkver.ps1:
```
> .\checkver -f office-tool-plus
office-tool-plus: 8.3.10.7 (scoop version is 8.3.10.7)
Forcing autoupdate!
Autoupdating office-tool-plus
Downloading Office_Tool_with_runtime_v8.3.10.7.7z to compute hashes!
Office_Tool_with_runtime_v8.3.10.7.7z (47.3 MB) [=============================================================] 100%
Computed hash: 757ac90af36ef1e92501429172a7670b884c1472be4eab99d42eb34de5a712a0
Writing updated office-tool-plus manifest
```

**Comment by issaclin32** [comment]
*persist* is not needed bacause config is at `HCKU\Software\OfficeToolPlus`.
(This is a reminder, no need to modify anything)

File: bucket/office-tool-plus.json (hunk 1 above)
Comment (ClassicDarkChocolate) [reply bucket/office-tool-plus.json]: Thanks!

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**